server/node.go

```diff
@@ -307,6 +307,7 @@ func (n *Node) bootstrapStores(bootstraps *list.List) {
 	for e := bootstraps.Front(); e != nil; e = e.Next() {
 		s := e.Value.(*storage.Store)
 		s.Bootstrap(sIdent)
+		s.Start()
```

Comment:
You should be checking the error returned by `Start`. And do you mind adding handling of the error returned by `Bootstrap`?